keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,23 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
-- kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
-  most machines
 - OpenAPI V3 is enabled by default on at least one version within kubectl's support window.
 As of Kubernetes 1.24 OpenAPIV3 entered beta and become enabled by default, therefore meeting this requirement.
+In Kubectl for release 1.25, all k8s versions within support window will be able to have OpenAPIV3 enabled.
+However, the fallback is kept around since it may not always be enabled.
```

Comment:
Mostly because I forgot, but can you remind me how the fallback work? Do we do it automatically or do we ask people to use the flag? DO we want to continue whatever we're doing?

Reply:
People can use the flag `--output plaintext-openapiv2` to directly use the openapiv2 backend without trying v3. Or, if fetching the V3 root document fails, the v2 document will be attempted while for the fallback

Reply:
Is this explained somewhere in the KEP? I'm having trouble finding it, and in fact the "OpenAPI V3 Not Available" seems to suggest that once we're defaulting the new flag, this condition will be an error.

Reply:
https://github.com/kubernetes/enhancements/blob/dea911f9a3f4035d766161e3938c63e2162046b8/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md#L461-L464

While there are versions of k8s within skew that only support beta OpenAPI we must have an automatic fallback to v2 for the defaulted flags.

However once OpenAPIV3 is GA there will be no k8s versions in skew which are allowed to disable it, and this would become an error condition. Until then we cannot treat "OpenAPI V3 Not Available"  as an error condition, unless the user explicitly requests to use openapiv3 via explicitly mentioning `--output plaintext`.

Reply:
Right, so isn't the main body of the KEP missing that explanation of what "fallback" means, maybe here?

https://github.com/kubernetes/enhancements/blob/dea911f9a3f4035d766161e3938c63e2162046b8/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md?plain=1#L348-L358

And doesn't this part need an update given that with this PR we're going to have a kubectl version with this feature on by default where a version within skew (1.26) does not have v3 in GA?

https://github.com/kubernetes/enhancements/blob/dea911f9a3f4035d766161e3938c63e2162046b8/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md?plain=1#L294-L315

Reply:
added